auth-api/src/auth_api/schemas/base_schema.py

```diff
@@ -33,22 +33,19 @@ class Meta:  # pylint: disable=too-few-public-methods
     modified_by = fields.Function(
         lambda obj: f'{obj.modified_by.firstname} {obj.modified_by.lastname}' if obj.modified_by else None
     )
+    exclude = ('versions')
```

Comment:
Excluding it in the base schema here, that way versions isn't queried when an object is serialized.